crates/bevy_ui/src/ui_node.rs

```diff
@@ -1064,6 +1064,30 @@ impl Overflow {
         }
     }
 
+    /// Hide overflowing items on both axes
```

Comment:
The comments needs to explain that hidden sections nodes are non-visible but still considered part of the layout (unlike clipped sections which are ignored completely).